projects/1.20/assets/under-the-moon/under_the_moon/lang/zh_cn.json

```
@@ -0,0 +1,302 @@
+{
+    "advancements.all_puffed_up.descr": "获得一颗马勃菌",
+    "advancements.all_puffed_up.title": "极度膨胀",
+    "advancements.blood_blast.descr": "用温迪戈的遗骸制作§4血咒图腾 §a",
+    "advancements.blood_blast.title": "血爆！",
+    "advancements.blood_for_the_blood_god.descr": "用温迪戈的遗骸制作§4恐惧利刃§a",
+    "advancements.blood_for_the_blood_god.title": "血祭血神",
+    "advancements.doesnt_play_in_the_circus.descr": "驯化一只烦乱狗首领，你可以借此命令其它烦乱狗而无需将其驯服",
+    "advancements.doesnt_play_in_the_circus.title": "别在马戏团表演",
+    "advancements.fetus_deletus.descr": "制作§4诅咒遗物§a...",
+    "advancements.fetus_deletus.title": "未出生的孩子",
+    "advancements.ill_be_your_jonny_on_the_spot.descr": "使用月光花和马勃菌来制作马勃菌汤",
+    "advancements.ill_be_your_jonny_on_the_spot.title": "我随叫随到",
+    "advancements.in_the_pines.descr": "进入蠕蔓森林",
+    "advancements.in_the_pines.title": "松林之中",
+    "advancements.is_it_alive.descr": "从战利品箱中获得诅咒橡子",
+    "advancements.is_it_alive.title": "它还活着？！",
+    "advancements.meet_the_beatles.descr": "破坏托运甲虫的巢穴入口",
+    "advancements.meet_the_beatles.title": "见见披头士乐队！",
+    "advancements.mooncharm.descr": "获得一颗§b月光石§a, 一种在战利品箱中产出的珍贵宝石",
+    "advancements.mooncharm.title": "月亮护符",
+    "advancements.moth_positive.descr": "制作一盏月灯",
+    "advancements.moth_positive.title": "飞蛾趋光",
+    "advancements.peek_no_more.descr": "击杀一只窥力怕，请小心它的自爆！",
+    "advancements.peek_no_more.title": "别偷窥了",
+    "advancements.pest_control.descr": "击杀一只回响虫",
+    "advancements.pest_control.title": "令人尴尬的蛐蛐叫...",
+    "advancements.quillbombed.descr": "制作一颗棘刺炸弹，作用不言而明...",
+    "advancements.quillbombed.title": "棘如轰炸",
+    "advancements.root.descr": "探索在每个人睡着后会出现的东西",
+    "advancements.root.title": "§b§l月亮之下",
+    "advancements.sends_quills_down_my_spine.descr": "击杀一只棘背兽，一种蠕蔓森林中的危险动物",
+    "advancements.sends_quills_down_my_spine.title": "从我的脊背下抛棘刺！",

```

```suggestion
    "advancements.sends_quills_down_my_spine.title": "寒意自棘背蔓延！",
```
这里似乎借鉴了send chills down spine
https://dictionary.cambridge.org/dictionary/english/send-chills-down-up-spine